.idea/misc.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why this change?

Reply:
Will revert it back to the original, I believe the misc.xml file automatically updated itself when I was working on changes in the other files.